src/main/java/com/health/controller/AjaxController.java

```diff
@@ -1189,6 +1189,98 @@ private HashMap<String, String> setResponse(Integer status) {
 
     }
 
+    @RequestMapping("/loadLanguageAndTopicByCategoryResource")
+    public @ResponseBody ArrayList<HashMap<Integer, String>> getLanguageByCategoryResourse(
+            @RequestParam(value = "catId") int catId) {
+
+        ArrayList<HashMap<Integer, String>> arlist = new ArrayList<>();
+        HashMap<Integer, String> languagesMap = new HashMap<>();
+        HashMap<Integer, String> topicMap = new HashMap<>();
+        HashMap<Integer, String> tutorialCount = new HashMap<>();
+
+        Category cat = catService.findByid(catId);
+        logger.info("Variables of loadLanguageByCategoryResourse cat : {}", cat);
+        if (cat != null) {
+            List<TopicCategoryMapping> tcm = topicCatService.findAllByCategory(cat);
+            List<ContributorAssignedTutorial> conList = conService.findAllByTopicCat(tcm);
+            List<Tutorial> tutList = tutService.findAllByconAssignedTutorialAndStatus(conList);
+
+            for (Tutorial tut : tutList) {
+                Category cat1 = tut.getConAssignedTutorial().getTopicCatId().getCat();
+                if (cat1.isStatus())
+                    languagesMap.put(tut.getConAssignedTutorial().getLan().getLanId(),
+                            tut.getConAssignedTutorial().getLan().getLangName());
+                topicMap.put(tut.getConAssignedTutorial().getTopicCatId().getTopic().getTopicId(),
+                        tut.getConAssignedTutorial().getTopicCatId().getTopic().getTopicName());
+
+            }
+
+            tutorialCount.put(tutList.size(), Integer.toString(tutList.size()));
+            arlist.add(languagesMap);
+            arlist.add(topicMap);
+            arlist.add(tutorialCount);
+        }
+
+        return arlist;
+
+    }
+
+    @RequestMapping("/loadTutorialCountByLanguage")
+    public @ResponseBody HashMap<Integer, String> getTutorialCountByLanguageResourse(
+            @RequestParam(value = "catId") int catId, @RequestParam(value = "lanIds[]") int[] lanIds) {
```

Comment:
```suggestion
    @RequestMapping("/loadTutorialCountByTopicAndLanguage")
    public @ResponseBody int getTutorialCountByTopicAndLanguage(
            @RequestParam(value = "catId") int catId, 
            @RequestParam(value = "topicIds[]") int[] topicIds,
            @RequestParam(value = "lanIds[]") int[] lanIds) {
```
Combine the two APIs

Change return type to int